### PR TITLE
Project accessability

### PR DESCRIPTION
**Pull request by suzi216**
**Web accessibility report** 

- Page title: `no issues found`
- Image text alternative: `Added the image text alternative`
- Text heading: `no issues found`
- Color contrast: `no issues found`
- Resize: `no issues found`
- Interaction: `no issues found`
- Moving content: `no issues found`
- Multimedia: `no issues found`
- Basic structure of the page: `no issues found`
- Links 'https://github.com/Suzi216/mobile_version_skeleton/blob/76d5d39d8f33f15a99e5b07837e4d82246ac8112/index.html#L160-L164'
https://github.com/Suzi216/mobile_version_skeleton/blob/76d5d39d8f33f15a99e5b07837e4d82246ac8112/index.html#L33-L37
 https://github.com/Suzi216/mobile_version_skeleton/blob/76d5d39d8f33f15a99e5b07837e4d82246ac8112/index.html#L51-L53
https://github.com/Suzi216/mobile_version_skeleton/blob/76d5d39d8f33f15a99e5b07837e4d82246ac8112/index.html#L59-L60

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
       <section class="navigator">
           <a href="#" class="mylogo">My logo</a>
           <nav>
```
Comment (t-yanick) [inline index.html:15]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md) for this project, " `Check that there is a title that adequately and briefly describes the content of the page.` ", please kindly update your page title to reflect the content of the page, currently it is `Skeleton project`, which is not descriptive of the page content. Checkout this [article](https://www.w3.org/WAI/test-evaluate/preliminary/#title) for more clarifications.

File: index.html [hunk 2]
```diff
@@ -30,34 +30,34 @@ <h1 class="head">I’m Suzana <br> Glad to see you!</h1>
...
-          <li class="i"><a href="#"><img src="images/angellist.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/twitter.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/github.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/medium.png" alt=""></a></li>
-          <li class="i"><a href="#"><img src="images/Linkedinicon.png" alt=""></a></li>
+          <li class="i"><a href="#"><img src="images/angellist.png" alt=" icons"></a></li>
+          <li class="i"><a href="#"><img src="images/twitter.png" alt="icons"></a></li>
+          <li class="i"><a href="#"><img src="images/github.png" alt="icons"></a></li>
+          <li class="i"><a href="#"><img src="images/medium.png" alt="icons"></a></li>
+          <li class="i"><a href="#"><img src="images/Linkedinicon.png" alt="icons"></a></li>
```
Comment (t-yanick) [inline index.html:37]: - [OPTIONAL] You should have added a link to this changes in the report summary  as specified in the requirements, checkout this [link](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148) on how to go about it.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @Suzi216, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**